xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/Help/Videos/WebHome.xml

```diff
@@ -42,65 +42,114 @@
 }, {
   'title': $services.localization.render('help.videos.videoCard4.title'),
   'topics': [
-    $services.localization.render('help.videos.videoCard4.topic1'),
-    $services.localization.render('help.videos.videoCard4.topic2')
+    {
+      'url': 'https://www.xwiki.org/xwiki/bin/view/Documentation/UserGuide/Features/Imports#HImportingOfficedocuments',
+      'label': $services.localization.render('help.videos.videoCard4.topic1')
+    },
+    {
+      'url': 'https://extensions.xwiki.org/xwiki/bin/view/Extension/TOC+Macro#HExample1:Simple',
+      'label': $services.localization.render('help.videos.videoCard4.topic2')
+    }
   ],
   'url': 'https://www.youtube.com/embed/csQVSRMVclM',
   'duration': '4 min'
 }, {
   'title': $services.localization.render('help.videos.videoCard5.title'),
   'topics': [
-    $services.localization.render('help.videos.videoCard5.topic1'),
-    $services.localization.render('help.videos.videoCard5.topic2')
+    {
+      'url': 'https://www.xwiki.org/xwiki/bin/view/Documentation/UserGuide/GettingStarted/CreatingABasicApp',
+      'label': $services.localization.render('help.videos.videoCard5.topic1')
+    },
+    {
+      'url': 'https://www.xwiki.org/xwiki/bin/view/Documentation/DevGuide/Tutorials/FAQTutorial/FAQTutorialAWM#HCreatingnewFAQentries',
+      'label': $services.localization.render('help.videos.videoCard5.topic2')
+    }
   ],
   'url': 'https://www.youtube.com/embed/Pv4jPCaU99g',
   'duration': '7 min'
 }, {
   'title': $services.localization.render('help.videos.videoCard6.title'),
   'topics': [
-    $services.localization.render('help.videos.videoCard6.topic1'),
-    $services.localization.render('help.videos.videoCard6.topic2')
+    {
+      'url': 'https://www.xwiki.org/xwiki/bin/view/Documentation/DevGuide/Tutorials/CreatingExtensions/#HInstallinganExtension',
+      'label': $services.localization.render('help.videos.videoCard6.topic1')
+    },
+    {
+      'url': 'https://extensions.xwiki.org/xwiki/bin/view/Extension/Administration%20Application#HDescription',
+      'label': $services.localization.render('help.videos.videoCard6.topic2')
+    }
   ],
   'url': 'https://www.youtube.com/embed/Q4NHu6J5pX4',
   'duration': '3 min'
 }])
 
 #macro (helpVideoCard $data)
   &lt;div class="videoCard"&gt;
-    &lt;iframe src="$data.url" allowfullscreen title="$escapetool.xml($data.title)" role="presentation" &gt;&lt;/iframe&gt;
+    &lt;iframe src="$data.url" allowfullscreen title="$escapetool.xml($data.title)" 
+        aria-description="$escapetool.html($services.localization.render('help.videos.hint'))" &gt;
+    &lt;/iframe&gt;
     &lt;div class="videoCard-body"&gt;
       &lt;div class="videoCard-title"&gt;
         $escapetool.xml($data.title)
       &lt;/div&gt;
       &lt;ul&gt;
         #foreach ($topic in $data.topics)
-          &lt;li&gt;$topic&lt;/li&gt;
+          &lt;li&gt;#if ($topic.url)
+            &lt;a href="$escapetool.html($topic.url)"&gt;$topic.label&lt;/a&gt;
```

Comment:
This needs to be `$escapetool.xml` as `$escapetool.html` doesn't escape `{`. Same for all other uses of `$escapetool.html`. Also, why is the label not escaped?

Reply:
> Also, why is the label not escaped?

I went too fast when updating the definition of the topics. I thought I'd factorize the escaping but forgot to add it back.

:+1: Addressed in https://github.com/xwiki/xwiki-platform/pull/2466/commits/61140f947991b0dd48234aefdd51ba1254bf42ec